packages/client/tests/functional/issues/12862-errors-are-obfuscated-by-interactive-transactions/tests.ts

```diff
@@ -25,7 +43,7 @@ testMatrix.setupTestSuite(
             viewCount: -1, // should fail, must be >= 0
           },
         }),
-      ).rejects.toThrow('violates check constraint \\"post_viewcount_check\\"')
```

Comment:
Should they be strictly equal?

Reply:
I think this is because errors are not implemented in pg adapter

Reply:
`toThrow` requires strict equivalent on substrings of the original error message, afaik

Reply:
Can be resolved so PR can be merged?

Reply:
I think the PR can be merged as is.